packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -257,7 +257,8 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
      */
     useEffect(() => {
         if (mapsindoorsSDKAvailable) {
-            const languageToUse = language ? language : navigator.language;
+            // Sets language to use. Priority: URL/Query Param -> App Config -> browser's default language.
+            const languageToUse = language ? language : appConfig?.appSettings?.language ? appConfig.appSettings.language : navigator.language;
```

Comment:
Can this be simplified using a nullish coalescing operator?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing